src/wovnio/wovnphp/Headers.php

```diff
@@ -346,4 +346,9 @@ public function shouldRedirect()
 
         return $cookieLang && ($requestLang !== $cookieLang) && ($requestLang === $this->store->defaultLang());
     }
+
+    public function isSearchEngineBot()
+    {
+        return strpos($this->env['HTTP_USER_AGENT'], 'Googlebot/') !== false;
```

Comment:
Could we try supporting more search engine:
- `bingbot/`
- `YandexBot/`
- `YandexWebmaster/`
- `DuckDuckBot-Https/`
- `Baiduspider/`

Reply:
Initially I wanted to add more support, but I just feel it's not useful to add other search engines because their market share is so small.

Reply:
I've added these bots to the bot list.

Reply:
Thanks